.travis.yml

```diff
@@ -1,11 +1,13 @@
 language: node_js
 node_js:
-  - '10.15.3'
-
+- 10.15.3
 script:
-  # the `audit-level` is at `moderate` to avoid https://www.npmjs.com/advisories/836, which has no resolution at time of writing
-  # consider swapping from `moderate` to `low` whenever this is possible and / reasonable (ex: when issue in the link above is fixed)
-  - npm audit --audit-level=moderate
-  - npm run flow
-  - npm run prettier
-  - npm run build
+- npm audit --audit-level=moderate
+- npm run flow
+- npm run prettier
+- npm run build
+env:
+  matrix:
+    secure: RBUY+C7OegZoLEAv0ukvFZlVQUN3PIHpNDiguJj66bBWVid8XHYRVLkmyxvN79VwR8zb5VPZmOHtFOnm6z5+AL9uCT/FEPBCJJ/uoglIszq1EW3zxv4pWRplrF25yM25QzfjmC9wy3GhQvpd9+OGDWPnXS0b2PjU2iC8Z+trJquXjN+maQ819cpeCAiw+gkmXhDWts/xSZK/OH2QhaEzhU1GBieFs8/mfhVIH84F3mPFPdoyoKsQqgwB1FK5FBCTvVAAkpjJnsbq2OZQebIWonrt5lplmGMxbNmh4xZpVffdMIJzSdr2l7sjrYpTjZkTvSOz+3vq7T1zK8+gn2Swpi0U1AZ18746QxUrPqfcuYxBk5BWnbHqYrjb28xVmAHen4jouUzT99g6tB1GMKmXNMMC2ey8WsxgBfh+JgOXhnzANOfTNZo0RMV/z0OZJp19LrFgfzvpmqiIdtraErT3ZpkJaIpNJaBZAOeqLvoBVdSeKDBtscPleW7HOMtteltFcN0l2l6R0fdl7gwkjvP1BP+sc0lFOEmzhzq16O736UeW8JfwbEGHyUXWkrnsC6pfsMhgQp4UBnclOlkMuCDSAZXlypY39vPMY1FNAAZGk6gce1kO132NBy2CWkt+Ibk/r+mXUsub2cVjBpAhtVKOvdGsbjOXzEm3eU29EXeWZZs=
```

Comment:
Two small blocking comments here:

- Can you add a comment saying something like "this is the encrypted lighthouse api key"
- Add add it to `env: global:` rather than `env: matrix`